Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "0.8.8"
+version = "0.8.9"
```

Comment:
Master is 0.8.9 now, so you will likely need to rebase and change that to 0.8.10